test/test_logger.rb

```diff
@@ -80,6 +72,36 @@ def test_basics
         generate_and_check_logfile
     end
 
+    def test_no_overwrite
+        task.overwrite_existing_files = false
+        task.auto_rename_existing_files = false
+        task.configure
+        touch_file = File.new(task.file, "w")
+        assert_raises Orocos::StateTransitionFailed do
+            task.start
+        end
+        puts '[INFO] This Error is expected and handled.'
+    end
+
+    def test_auto_renaming
+        task.overwrite_existing_files = false
+        task.auto_rename_existing_files = true
+        touch_file = File.new(task.file, "w")
+        assert(!task.has_port?('time'))
+        assert(task.createLoggingPort('time', '/base/Time', []))
+        generate_and_check_logfile
+    end
+
+    def test_ambiguous_properties
+        task.overwrite_existing_files = true
+        task.auto_rename_existing_files = true
+        task.configure
+        assert_raises Orocos::StateTransitionFailed do
+            task.start
+        end
+        puts '[INFO] This Error is expected and handled.'
```

Comment:
Don't puts in tests. If you want something informative, make the test name more explicit and run with `-v` to see the test progress.